test/test.js

```diff
@@ -25,11 +27,27 @@ var qunitRunner = new JsReporters.QUnitAdapter(QUnit)
 var qunitTestReporter = new JsReporters.TestReporter(qunitRunner, referenceData.QUnit)
 
 QUnit.done(function () {
-  process.exit(qunitTestReporter.ok ? 0 : 1)
+  if (!qunitTestReporter.ok) {
+    process.exit(1)
+  }
 })
 
 QUnit.config.autorun = false
 
 require('./qunit/tests.js')
 
 QUnit.load()
+
+var mocha = new Mocha()
+var mochaRunner = new JsReporters.MochaAdapter(mocha)
+var mochaTestReporter = new JsReporters.TestReporter(mochaRunner, referenceData.Mocha)
+
+mocha.addFile(
+  path.join(__dirname, 'mocha/tests.js')
```

Comment:
Which one above? Do you mean this [one](https://github.com/js-reporters/js-reporters/pull/46/files?diff=unified#diff-cbe814bd39e9f048a8d19ed4d2006bd5R46). I think it can take also `asolute paths` as also `relative to the current directory`.